src/test/resources/features/Meeds/MeedsProfile.feature

```diff
@@ -134,7 +135,7 @@ Feature: Search for User Informations in Profile page
     And I remove my profile work experiences 'QA Engineer'
     And I remove my profile work experiences 'IT Engineer Developer'
 
-  @ignore
+  @profile
```

Comment:
Add `@profile` on top of the file once for all declared scenarios here

Reply:
done